src/bin/pg_autoctl/keeper_config.c

```diff
@@ -195,6 +195,7 @@
 		OPTION_POSTGRESQL_PROXY_PORT(config), \
 		OPTION_POSTGRESQL_LISTEN_ADDRESSES(config), \
 		OPTION_POSTGRESQL_AUTH_METHOD(config), \
+		OPTION_POSTGRESQL_HBA_LEVEL(config), \
```

Comment:
If I understand this change correctly it means we didn't write hba-level to the ini file before. How will this change interact with nodes created with a previous version of pg_auto_ctl that were created with `--skip-pg-hba`? What will users need to do to fix these nodes?

Reply:
When the configuration file was not updated, we would fill-in the gap with using `minimal` at read-time. To fix it is possible to use `pg_autoctl set config postgresql.hba_level skip`. I don't see how to automate that, so I believe we should address the situation with a special release note at the next release?

Reply:
Okay, makes sense. I already thought we couldn't automatically fix this for existing users. Let's make a clear note of this in the next release then.